kuma/wiki/templates/wiki/revision.html

```diff
@@ -8,7 +8,7 @@
 {% block content %}
 
   <article>
-    <h1>{{ document.title }}</h1>
+    <h1><a href="{{ url('wiki.document', document.full_path, locale=document.locale) }}">{{ _('{title}')|fe(title=document.title) }}</a></h1>
```

Comment:
Okay, I told you I always get confused with this. :smile:

Both ways are safe on a local spot-check: `{{ document.title }}` and `{{ _('{title}')|fe(title=document.title) }}` with a title of `<script>alert("ouch")</script>` because [we've enabled Jinja autoescaping](https://github.com/mozilla/kuma/blob/master/settings.py#L559). A search in the code shows we use both methods thru-out, so there's no tie-breaker there. :frowning:

With that said, I prefer this `|fe` usage because it's more explicit.

@robhudson - I think yours is un-safe because you did `title|fe(title=title)` and not `'{title}'|fe(title=title)` ? I.e., you didn't name the var for string interpolation?